PacketDetection/ScannerRegistry.cs

```diff
@@ -339,7 +341,7 @@ private void DeclareScanners()
                                BitConverter.ToUInt64(packet.Data, Offsets.IpcData + 0x18) != 0 &&
                                BitConverter.ToUInt32(packet.Data, packet.Data.Length - 4) == 0);
             //=================
-            RegisterScanner("PrepareZoning", "Please find an Aetheryte and teleport to Mih Khetto's Amphitheatre.",
+            RegisterScanner("PrepareZoning", "Please find an Aetheryte and teleport to Lancers' Guild.",
```

Comment:
Is there a reason why this location had to be changed? PrepareZoning is sent before ActorSetPos, so if the two scanners involve the same aetheryte location they can be resolved in one action.

Reply:
This is fine actually, it should work the same way regardless of what the directions are. No worries here.